programs/liquidity-incentive-program/src/instructions/end_deposit.rs

```diff
@@ -101,7 +104,8 @@ pub fn process(ctx: Context<EndDeposit>) -> Result<()> {
 
     // Transfer any additional rewards to the ephemeral token account
     if additional_reward_amount > 0 {
-        transfer(
+        #[allow(deprecated)]
+        anchor_spl::token_2022::transfer(
```

Comment:
So all calls to a token program are moved to using the token22 fns, because they handle both programs?

Reply:
Yes, exactly, both instructions should actually be the same data wise, the difference is just that the old `transfer` allows only tokenkeg, while `token_2022::transfer` allows both programs